google-cloud-phishing_protection-v1beta1/lib/google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "gapic/common"
+require "gapic/config"
+require "gapic/config/method"
+
+require "google/cloud/phishing_protection/v1beta1/version"
+
+require "google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1/credentials"
+require "google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1/paths"
+require "google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1/client"
+
+module Google
+  module Cloud
+    module PhishingProtection
+      module V1beta1
+        ##
+        # Service to report phishing URIs.
+        #
+        # To load this service and instantiate a client:
+        #
+        #     require "google/cloud/phishing_protection/v1beta1/phishing_protection_service_v1_beta1"
+        #     client = Google::Cloud::PhishingProtection::V1beta1::PhishingProtectionServiceV1Beta1::Client.new
+        #
+        module PhishingProtectionServiceV1Beta1
```

Comment:
The name of this service (`PhishingProtectionServiceV1Beta1`) doesn't really follow the typical convention. Shouldn't it be just `PhishingProtectionService`? Is this worth investigating upstream?

Reply:
Thanks for catching that. The service name in the upstream proto is indeed `PhishingProtectionServiceV1Beta1` which is terrible, but cannot be changed. I'll look into mapping our class names to something a bit better.

Reply:
phixed.